pkgs/development/libraries/gobject-introspection/setup-hook.sh

```diff
@@ -10,7 +10,17 @@ make_gobject_introspection_find_gir_files() {
     fi
 }
 
+export_gobject_introspection_fallback_path() {
+    # Newer multiple-output-optimized stdenv has an environment variable
```

Comment:
In the context of structured attrs "environment variable" is not correct, because it's precisely not an environment variable, but just a bash variable.

Reply:
That's what I get for copying the old comment :) Tweaked it.